packages/terra-site/webpack.config.js

```diff
@@ -70,6 +71,7 @@ module.exports = {
       log: false,
       plugins: [
         PostCSSCustomProperties({ preserve: true }),
+        rtl(),
```

Comment:
id' rather we move PostCSSCustomProperties into `postcss.config.js` if possible

Reply:
Thanks for ignoring my statement. The reasoning behind a consolidated postcss config is to make it reusable for storybook in the future as well as easier for consumers to see/reuse that config.

Reply:
I apologize, I realized this was merged before my comment. I do suggest we consolidate/simplify the config